video/out/w32_common.c

```diff
@@ -1055,6 +1055,32 @@ static void update_dark_mode(const struct vo_w32_state *w32)
     DwmSetWindowAttribute(w32->window, DWMWA_USE_IMMERSIVE_DARK_MODE,
                           &use_dark_mode, sizeof(use_dark_mode));
 }
+#if __MINGW64_VERSION_MAJOR >= 12
```

Comment:
I would prefer to copy needed definition into mpv. This is generaly done for older mingw.

Reply:
Yeah actually if you just copy what #11469 that would work just fine.

Reply:
And then you can remove mapping `intToBackdrop` function